netbout-client/pom.xml

```diff
@@ -180,7 +180,13 @@
                         <artifactId>qulice-maven-plugin</artifactId>
                         <configuration>
                             <excludes>
-                                <exclude>findbugs:.*</exclude>
+                                <!--
+                                @todo #911:30min Fix findbugs errors in client
+                                 Some issues may not require fixing an should be
+                                 excluded. Filters in qulice 0.12 are broken, right
```

Comment:
@exper0 let's merge this PR, please update the puzzle
I think it should be resolved when #961 completed due to broken Findbugs filters in current  version of Qulice